daml-lf/validation/BUILD.bazel

```
@@ -13,10 +13,7 @@ da_scala_library(
     srcs = glob(["src/main/**/*.scala"]),
     scalacopts = lf_scalacopts,
     tags = ["maven_coordinates=com.daml:daml-lf-validation:__VERSION__"],
-    visibility = [
-        "//compiler/scenario-service:__subpackages__",
-        "//daml-lf:__subpackages__",
-    ],
+    visibility = ["//visibility:public"],
```

```suggestion
    visibility = ["//:__subpackages__"],
```
Unless you are interested in this being globally visible.

---

As part of #7399, each ledger implementation will have to validate packages before committing them.
So they will need access to this packages.